### PR TITLE
feat(gallery): dir2json now parses imports

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

### CODE REVIEW

File: demos/gallery/util/dir2json.js [hunk 1]
```diff
@@ -19,9 +19,32 @@ function dirTree(filename) {
...
+    function _parseImports(filename){
+        let dep_obj = {};
+        const file_str = fs.readFileSync(filename).toString();
+        let import_arr = file_str.split('import ').map(imp=>{
+            const sp2 = imp.split(' from ');
+            if(sp2 && sp2[1]){
+                const src_name = sp2[1].split('\n')[0]
+                .replace('"','')
+                .replace('"','')
+                .replace(';','')
```
Comment (GordonSmith) [inline demos/gallery/util/dir2json.js]: indentation

File: demos/gallery/util/dir2json.js [hunk 2]
```diff
@@ -19,9 +19,32 @@ function dirTree(filename) {
...
+                ;
+                const dep_names = sp2[0]
+                .replace('{','')
+                .replace('}','')
+                .split(',')
```
Comment (GordonSmith) [inline demos/gallery/util/dir2json.js]: indentation

**Comment by jbrundage** [comment]
@GordonSmith indentation corrected

File: demos/gallery/util/dir2json.js [hunk 3]
```diff
@@ -19,9 +19,32 @@ function dirTree(filename) {
...
     }
 
     return info;
+
+    function _parseImports(filename){
+        let dep_obj = {};
+        const file_str = fs.readFileSync(filename).toString();
+        let import_arr = file_str.split('import ').map(imp=>{
+            const sp2 = imp.split(' from ');
+            if(sp2 && sp2[1]){
```
Comment (GordonSmith) [inline demos/gallery/util/dir2json.js]: sp2 will always be defined?

Comment (GordonSmith) [reply demos/gallery/util/dir2json.js]: I suspect you meant to test: sp2.length > 2?

**Comment by jbrundage** [comment]
@GordonSmith fixed

**Pull request merged**